_posts/2020-12-22-ols.md

```diff
@@ -0,0 +1,76 @@
+
+## [Growing the Galaxy Community](https://openlifesci.org/ols-2/projects-participants/#growing-the-galaxy-community), leaded by [Beatriz Serrano-Solano](https://openlifesci.org/ols-2/projects-participants#beatrizserrano) and mentored by [Dave Clements](https://openlifesci.org/ols-2#tnabtaf)
+
+The Galaxy community (users, developers, admins...) is growing all over the world and reaching new scientific fields. Time zones, and the growing number of work to manage such a big community, needs the help of a community manager in Europe working in close collaboration with other community managers around the world.
+
+At the beginning of September 2020, Beatriz started a new position at the University of Freiburg to help with the community management within the European Galaxy team. Along the 15 weeks of the program, she had the opportunity to advance her community building and project management skill while enhancing the connection with the different communities inside and outside Europe.
+
+You can watch her graduation presentation there:
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/wv8_0iOMpGs?start=1639" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## [Creating a single pipeline for metagenome classification](https://openlifesci.org/ols-2/projects-participants/#creating-a-single-pipeline-for-metagenome-classification), leaded by [Muhammet Celik](https://openlifesci.org/ols-2/projects-participants#MuhammetCelik) and mentored by [Mallory Freeberg](https://openlifesci.org/ols-2#malloryfreeberg)
+
+Metagenomics has been increasingly becoming very important in studies of human and animal health. It has become clear that bacteria in and on our bodies are very significant. Thus there is a big boost in science society who are willing to study metagenomics. When we want to do downstream analysis in our microbial community, the first step is to find out about our community, what the community ultimately looks like.
```

Comment:
Metagenomics is increasingly important in ...

Thus there is a big boost in the number of scientists who are studying metagenomics. ...